crates/test_utils/src/minicore.rs

```diff
@@ -31,6 +31,7 @@
 //!     eq: sized
 //!     ord: eq, option
 //!     derive:
+//!     fmt:
```

Comment:
```suggestion
//!     fmt: result
```
Since this has a type alias on `Result` we should include the dependency on it, didn't see that earlier sorry.
Otherwise lgtm!
bors d+